apps/aecore/src/aec_metrics_lib.erl

```diff
@@ -1,16 +1,30 @@
 -module(aec_metrics_lib).
 
 -export([find_entries_cmd/1,
+         find_entries_cmd/2,
+         filter_entries/2,
          get_values_cmd/1]).
 
 find_entries_cmd(Arg) ->
-    find_entries_cmd(Arg, enabled).
+    find_entries_(Arg, enabled, fun exometer:select/1).
 
 find_entries_cmd(Arg, Status) ->
+    find_entries_(Arg, Status, fun exometer:select/1).
+
+filter_entries(Entries, Arg) ->
+    Entries = [{N,D,S} || {N,D,S} <- Entries,
```

Comment:
This line silently discards items in `Entries` that do not match `{_,_,_}`, making it fragile to future code changes. May I suggest replacing this with code that hardcodes the expectation on each item of `Entries`.

Reply:
This gave me reason to double-check the implementation, and there are actually a few more things I'd like to correct. Will push a new version a bit later.